Readme.md

```diff
@@ -58,10 +58,11 @@ memory_limit=128M
 | Imagick            | `${bref:extra.imagick-php-74}`        |
 | LDAP               | `${bref:extra.ldap-php-74}`           |
 | Mailparse          | `${bref:extra.mailparse-php-74}`      |
-| Memcache           | `${bref:extra.memcache-php-74}`      |
+| Memcache           | `${bref:extra.memcache-php-74}`       |
 | Memcached          | `${bref:extra.memcached-php-74}`      |
 | MongoDB            | `${bref:extra.mongodb-php-74}`        |
 | MsgPack            | `${bref:extra.msgpack-php-74}`        |
+| Newrelic           | `${bref:extra.newrelic-php-74}	`      |
```

Comment:
remove extra space to align `|` (OCD review :P)

Reply:
Don't worry about reviewing for the moment. Something in the newest bref layer confuses the PHP Agent installer into thinking it's ZTS compiled and our installer is putting the ZTS version of our agent in instead of the non-ZTS.

I can manually pull the non-ZTS version but I want to see if there's a way to over-ride the installer to avoid having to put checks of PHP API into the Dockerfile if it's not neccessary. I'm hoping to figure this out over the coming days and get another push to you.

Reply:
OK I've resolved all that. Build looked good locally now. Tested with a basic app again and working and reporting fine. So hopefully this will be the last rebase. Rebased as changelog 0.9.4 config and test file are included this time. Sorry for the back and forth on this one.